src/main/java/com/linkedin/replica/chat/commands/impl/GetHistoryCommand.java

```diff
@@ -0,0 +1,36 @@
+package com.linkedin.replica.chat.commands.impl;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+import com.linkedin.replica.chat.commands.Command;
+import com.linkedin.replica.chat.database.handlers.ChatHandler;
+
+public class GetHistoryCommand extends Command {
+
+	public GetHistoryCommand(HashMap<String, Object> args) {
+		super(args);
+	}
+
+
+	@Override
+	public String execute(){
+		validateArgs(new String[]{"userId1", "userId2"});
+
+		String userId1 = (String) args.get("userId1");
+		String userId2 = (String)args.get("userId2");
+		String limit = (String)args.get("limit");
+		String offset = (String)args.get("offset");
+
+		ChatHandler chatHandler = (ChatHandler) this.dbHandler;
+
+		if ((limit.equals(null) && offset.equals(null) || (limit.isEmpty() && offset.isEmpty()))) {
```

Comment:
Please handle the case where one of the offset or limit is null to avoid NullPointerException in else clause.